docs/how-to/gateway-best-practices.md

```diff
@@ -35,7 +35,7 @@ If you are running an IPFS node that is also configured as an IPFS gateway, each
 - Put a CDN like Cloudflare in front of the IPFS gateway.
 - Consider enabling the [Accelerated DHT Client](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#accelerated-dht-client). 
 - Test and monitor your internet connection speed, with a tool like [Speedtest CLI](https://www.speedtest.net/apps/cli).
-- Monitor disk I/O and make sure that no other processes are causing disk I/O bottlenecks with a tool like [iotop](https://linux.die.net/man/1/iotop) or [iostat](https://linux.die.net/man/1/iostat).
```

Comment:
Why did these links change and why is one to GitHub and the other not (looks like iostat might be here https://github.com/sysstat/sysstat)?

Reply:
`linux.die.net`  return 403 to requests coming from CI, likely some overzealous abuse protection. I'll switch the iostat to github too, thanks!